### DISCUSSION TITLE
View/Copy as JSON

### DISCUSSION THREAD

**Discussion by hwillson** [apollographql/apollo-client-devtools#387] [Ideas]
Migrated from: https://github.com/apollographql/apollo-client-devtools/issues/258

Hey Hey

Thanks for making this great extension, it really helps my workflow and is a big part of the value proposition of apollo client

-----

I'd love to be able to view and/or copy an object from the cache inspector as JSON. That would help me for instance in writing test cases.

Thanks again

**Comment by jerelmiller** [comment]
I believe this is implemented with the copy button in the top-right of the Cache tab:

<img width="1512" alt="Screenshot 2024-06-06 at 12 03 40 PM" src="https://github.com/apollographql/apollo-client-devtools/assets/565661/4c9b041a-1b9c-4c1a-a2f1-c73624732124">

As such, I'm going to go ahead and close this out 🙂
